### PR TITLE
Updates analytics users

### PR DESCRIPTION
**Pull request by Baalmart**
- What does this PR do?
This PR just updates the endpoint for `Default Settings`. Slightly modifies the `Welcome email message` to be more descriptive. And it passes all the  unit tests for `Forgot Password` and` Reset Password`.
- What JIRA task is related to this PR?
No card
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
Base url: http://localhost:3000/api/v1/users

> partial url: /forgotPassword
> partial url: /reset/you/<TOKEN>
> partial url: /defaults/<USER ID>
> partial url: /defaults/<USER ID>
> partial url: /updatePassword
> partial url: /updatePasswordViaEmail
> partial url: /registerUser

Detailed notes of these endpoints can be found in the [AirQo API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=2081265667)

### CODE REVIEW

**Comment by danielogen** [comment]
Hi @Baalmart , I'm getting this error: `unenable to send email`.
`sending mail
there was an error:  { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt
534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu
534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv>
534-5.7.14 Please log in via your web browser and then try again.
534-5.7.14  Learn more at`

Do I need to change anything in the environment file?

**Comment by Baalmart** [comment]
> Hi @Baalmart , I'm getting this error: `unenable to send email`.
> `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> 
> Do I need to change anything in the environment file?

I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....

**Comment by danielogen** [comment]
> > Hi @Baalmart , I'm getting this error: `unenable to send email`.
> > `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> > Do I need to change anything in the environment file?
> 
> I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....

Thanks for the environmental variable file. however, I'm still not able to get around this.

**Comment by danielogen** [comment]
> > > Hi @Baalmart , I'm getting this error: `unenable to send email`.
> > > `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> > > Do I need to change anything in the environment file?
> > 
> > 
> > I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....
> 
> Thanks for the environmental variable file. however, I'm still not able to get around this.

Screenshot 2020-06-17 at 19 24 06
* The console message

**Comment by Baalmart** [comment]
> > > > Hi @Baalmart , I'm getting this error: `unenable to send email`.
> > > > `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> > > > Do I need to change anything in the environment file?
> > > 
> > > 
> > > I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....
> > 
> > 
> > Thanks for the environmental variable file. however, I'm still not able to get around this.
> 
> Screenshot 2020-06-17 at 19 24 06
> 
> * The console message

let me try to reproduce...

**Comment by Baalmart** [comment]
> > > > > Hi @Baalmart , I'm getting this error: `unenable to send email`.
> > > > > `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> > > > > Do I need to change anything in the environment file?
> > > > 
> > > > 
> > > > I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....
> > > 
> > > 
> > > Thanks for the environmental variable file. however, I'm still not able to get around this.
> > 
> > 
> > Screenshot 2020-06-17 at 19 24 06
> > 
> > * The console message
> 
> let me try to reproduce...

Things are working on my side....strange, right? What could we be missing?
image

**Comment by Baalmart** [comment]
> > > > > > Hi @Baalmart , I'm getting this error: `unenable to send email`.
> > > > > > `sending mail there was an error: { Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbt 534-5.7.14 IH-vBQL3ETO95Tmry5BA5TBUqAcIiK40X3XTxyM7NmC1CKwFfoZo7czff7_nQ4dE5rSdu 534-5.7.14 MLwKEEojQzDqbdabs2A_OdACGdc6xoQeeaSV0zb1Y8NxmeMXTYqFTZREgglnkuYv> 534-5.7.14 Please log in via your web browser and then try again. 534-5.7.14 Learn more at`
> > > > > > Do I need to change anything in the environment file?
> > > > > 
> > > > > 
> > > > > I am not sure - first time I am seeing this error - never seen it before. I think we need to crosscheck the environmental variable file. Please check your Slack DM in a few seconds from now....
> > > > 
> > > > 
> > > > Thanks for the environmental variable file. however, I'm still not able to get around this.
> > > 
> > > 
> > > Screenshot 2020-06-17 at 19 24 06
> > > 
> > > * The console message
> > 
> > 
> > let me try to reproduce...
> 
> Things are working on my side....strange, right? What could we be missing?
> image

A quick one, is that email "whose password was forgotten" registered? What response message are you getting from the endpoint?

**Comment by danielogen** [comment]
Hi @Baalmart: This is exactly the issue I'm facing https://github.com/nodemailer/nodemailer/issues/998.  To reproduce: Git clone this repo `cd AirQo-api/src/auth-service`, `npm install` and `npm run dev`. Now try to register a new user following the [API doc](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=2081265667) guide.

File: src/data-mgt/.gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ lib-cov
 *.out
```
Comment (danielogen) [inline src/data-mgt/.gitignore:5]: This file seems to be coming from another microservice `data-mgt`.

**Pull request merged**